src/components/organelle-wrapper.tsx

```diff
@@ -78,30 +78,31 @@ class OrganelleWrapper extends React.Component<OrganelleWrapperProps, OrganelleW
       height: 312
     };
 
-    modelDef.properties = modelProperties;
+    modelDef.properties = modelProperties.toJS();
```

Comment:
Only downside is that we need to do things like this to convert an `ObservableMap` back to an ES6 map.